Modules/Office365DSC/DSCResources/MSFT_SCComplianceSearchAction/MSFT_SCComplianceSearchAction.schema.mof

```diff
@@ -0,0 +1,15 @@
+[ClassVersion("1.0.0.0"), FriendlyName("SCComplianceSearchAction")]
+class MSFT_SCComplianceSearchAction : OMI_BaseResource
+{
+    [Key, Description("The Action parameter specifies what type of action to define. Accepted values are Export and Purge."), ValueMap{"Export","Purge","Retention"}, Values{"Export","Purge","Retention"}] String Action;
```

Comment:
Either add "Retention" to the description as an accepted value or remove it from the valuemap.